deps/curl-8.1.2/lib/vtls/sf_ocsp.c

```diff
@@ -711,9 +711,31 @@ static OCSP_RESPONSE * queryResponderUsingCurl(char *url, OCSP_CERTID *certid, c
         sf_otd_set_event_sub_type(OCSP_RESPONSE_ENCODE_FAILURE, ocsp_log_data);
         goto end;
       }
+
+      // send the entire OCSP URL to the cache server
+      char full_url[4096] = "";
+      strcpy(full_url, host ? host : "");
+
+      // default port numbers will not be added to the url
+      // (443 if the scheme is https, else 80)
+      if (port &&
+          atoi(port) > 0 &&
+          ((use_ssl && atoi(port) != 443) || (!use_ssl && atoi(port) != 80)))
+    {
+        strcat(full_url, ":");
+        strcat(full_url, port);
+    }
+
+      // path is guaranteed to begin with a / character
+      if (path && (strlen(path) > 1))
+      {
+          strcat(full_url, path);
```

Comment:
Shouldn't we URL encode path? Does our `retry` proxy handle correctly slashes?

Reply:
Support team has confirmed the fix already so URL encoded is not required.